### DISCUSSION TITLE
stitching.stitching_error.StitchingError: Camera parameters adjusting failed.

### DISCUSSION THREAD

**Discussion by a514154639** [OpenStitching/stitching#108] [Q&A]
I want to stitch a track picture as follows
![image](https://github.com/OpenStitching/stitching/assets/39019284/1e73f6ff-3fd8-4860-9e6d-b1ec4e26a89c)
but when I use  stitch * directly, an error show that Camera parameters adjusting failed.
then I use " stitch * --adjuster no" it works, but the result seems not good.
![image](https://github.com/OpenStitching/stitching/assets/39019284/d4b89087-70e9-472a-9687-ec3db3e1e544)
I will appreciate it if you can tell me how to solve this problem.

**Comment by lukasalexanderweber** [comment]
try with --affine

**Reply by a514154639** [reply]
![image](https://github.com/OpenStitching/stitching/assets/39019284/c80bf79a-a32c-4db2-bb42-837408a65fff)
seems not good

**Comment by lukasalexanderweber** [comment]
Can you please follow the tutorial to check at which point it starts not working?

**Reply by lukasalexanderweber** [reply]
Also check out https://github.com/OpenStitching/stitching/discussions/81

**Reply by lukasalexanderweber** [reply]
Any updates? It seems like an interesting use case if you can share the images
